src/main/java/org/hobbit/utils/kubernetes/KubeHelper.java

```diff
@@ -0,0 +1,40 @@
+package org.hobbit.utils.kubernetes;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+
+public class KubeHelper {
```

Comment:
`DNSHelper`

Reply:
Also, I add DNSHelper asa  class to use by modules and that class will use this Kubernetes class, then I keep the name